ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -248,6 +248,12 @@ func validateRayServiceSpec(rayService *rayv1.RayService) error {
 	if headSvc := rayService.Spec.RayClusterSpec.HeadGroupSpec.HeadService; headSvc != nil && headSvc.Name != "" {
 		return fmt.Errorf("spec.rayClusterConfig.headGroupSpec.headService.metadata.name should not be set")
 	}
+
+	if currStrat := rayService.Spec.UpgradeStrategy; currStrat != "" {
+		if currStrat != rayv1.NewCluster && currStrat != rayv1.None {
+			return fmt.Errorf("spec.UpgradeStrategy is invalid")
```

Comment:
Can you print the invalid value and the valid options?

Reply:
Done!